imagebuildah/stage_executor.go

```diff
@@ -867,6 +868,21 @@ func (s *StageExecutor) getImageRootfs(ctx context.Context, image string) (mount
 	return builder.MountPoint, nil
 }
 
+// getContentSummary generates content summary for cases where we added content and need
+// to get summary with updated digests.
+func (s *StageExecutor) getContentSummaryAfterAddingContent() string {
+	addedContentType, addedContentDigest := s.builder.ContentDigester.Digest()
```

Comment:
(Absolutely nonblocking: In this now much smaller function, this can use shorter identifiers, e.g.`type`/`digest`/`summary`.)